airflow/providers/amazon/aws/triggers/batch.py

```diff
@@ -105,3 +105,92 @@ async def run(self):
             yield TriggerEvent({"status": "failure", "message": "Job Failed - max attempts reached."})
         else:
             yield TriggerEvent({"status": "success", "job_id": self.job_id})
+
+
+class BatchSensorTrigger(BaseTrigger):
+    """
+    Checks for the status of a submitted job_id to AWS Batch until it reaches a failure or a success state.
+    BatchSensorTrigger is fired as deferred class with params to poll the job state in Triggerer.
+
+    :param job_id: the job ID, to poll for job completion or not
+    :param aws_conn_id: connection id of AWS credentials / region name. If None,
```

Comment:
These are out of order.

Reply:
Fixed in [388cbaa](https://github.com/apache/airflow/pull/30279/commits/388cbaa3572fd7e46985e33edadd92ec372d7169)